psd-web/db/seeds.rb

```diff
@@ -413,36 +413,68 @@ def create_blob(filename, title: nil, description: nil)
 
   investigation.products << product
 
-  organisation = Organisation.create!(name: "Office for Product Safety and Standards")
-
-  enforcement = Team.create!(name: "OPSS Enforcement", team_recipient_email: "enforcement@example.com", "organisation": organisation)
-  processing  = Team.create!(name: "OPSS Processing", team_recipient_email: nil, "organisation": organisation)
-
-  Team.create!(name: "OPSS Science and Tech", team_recipient_email: nil, "organisation": organisation)
-  Team.create!(name: "OPSS Trading Standards Co-ordination", team_recipient_email: nil, "organisation": organisation)
-  Team.create!(name: "OPSS Incident Management",  team_recipient_email: nil, "organisation": organisation)
-  Team.create!(name: "OPSS Testing", team_recipient_email: nil, "organisation": organisation)
-
-  # On review apps environment (int) this will be set using `psd-seed-users-env`
-  # example of variable:
-  # SEED_USERS='JohannMuster:muster@example.org:07777888999;JohnDoe:example@example.org:07777888999'
-  if ENV["SEED_USERS"]
-    ENV["SEED_USERS"].split(";").map { |set| set.split(":") }.each do |name, email, phone|
-      user = User.create!(
-        name: name,
-        email: email,
-        password: "testpassword",
-        password_confirmation: "testpassword",
-        organisation: organisation,
-        mobile_number_verified: true,
-        teams: [enforcement],
-        mobile_number: phone
-      )
-      %i[superuser team_admin opss_user psd_user user].each do |role|
-        UserRole.create!(user: user, name: role)
+  if Rails.env.production? && (organisations = CF::App::Credentials.find_by_service_name("psd-seeds")["organisations"]) # rubocop:disable Rails/DynamicFindBy
```

Comment:
Please change: This will throw an exception when the service isn't defined (on staging/prod): ```NoMethodError (undefined method `[]' for nil:NilClass)```
```suggestion
  if Rails.env.production? && (organisations = CF::App::Credentials.find_by_service_name("psd-seeds").try(:[], "organisations")) # rubocop:disable Rails/DynamicFindBy
```